modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -775,6 +775,9 @@ private void generateSupportingFiles(List<File> files, Map<String, Object> bundl
         bundle.put("basePathWithoutHost", basePathWithoutHost);
         bundle.put("scheme", URLPathUtils.getScheme(url, config));
         bundle.put("host", url.getHost());
+        if (url.getPort() != 80 ) {
```

Comment:
@wing328 shall we consider 443 for HTTPS too?

Reply:
Sounds good. Do you have time to contribute a PR with your suggestion?

Reply:
I've filed https://github.com/OpenAPITools/openapi-generator/pull/7531